tests/file_uploads/tests.py

```diff
@@ -38,6 +40,10 @@
     '../..\\hax0rd.txt',        # Relative path, mixed.
     '..&#x2F;hax0rd.txt',       # HTML entities.
     '..&sol;hax0rd.txt',        # HTML entities.
+    'hax><:?|*0rd.txt',         # with reserved characters.
+    'hax><:?|*0rd..txt',        # file name ends with a period.
```

Comment:
I'm also not sure why a file ended with `..txt` is an issue :thinking:

Reply:
From the documentation I mentioned above, it states "Do not end a file or directory name with a space or a period. Although the underlying file system may support such names, the Windows shell and user interface does not. However, it is acceptable to specify a period as the first character of a name. For example, ".temp"."
this was the reason behind adding those extra checks.